spec/support/server.rb

```diff
@@ -47,18 +48,35 @@ def last_received_report
       end
     end
 
+    def auth_response_code
+      @mutex.synchronize do
+        @auth_esponse_code
```

Comment:
```suggestion
        @auth_response_code
```

(also in `auth_response_code` and `clear!` below)

Reply:
aaaaaaaa I've fixed that three times by now 🤦